tests/unit/Makefile.am

```diff
@@ -7,8 +7,9 @@ check_SCRIPTS += \
 	test_package_module_yum
 endif
 
-if HAVE_PKG_INSTALL
 if HAVE_SHUNIT2
+check_SCRIPTS += test_package_module_freebsd_ports
+if HAVE_PKG_INSTALL
```

Comment:
We only need `shunit2` available to run the tests - all the pkg / ports stuff is stubbed or mocked :)